### PR TITLE
Update unzip to 1.1

### PR DESCRIPTION
**Pull request by moradology**

### CODE REVIEW

**Comment by moradology** [comment]
https://launchpad.net/ubuntu/+source/unzip/6.0-9ubuntu1.1

**Comment by notthatbreezy** [comment]
I think the apt command needs to have an `update_cache=yes` in `tasks/main.yml` else this fails in the examples folder.

File: tasks/main.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ---
 - name: Install unzip
-  apt: pkg=unzip={{ unzip_version }} state=present
+  apt: pkg=unzip={{ unzip_version }} state=present update_cache=yes
```
Comment (hectcastro) [inline tasks/main.yml]: I think `update_cache` should be removed from here and instead go in the `site.yml` in the `examples` directory. If we keep it here, anyone who uses this role will incur the wait associated with an `apt-get update` every time this task gets executed.

Example:

``` yaml
---
- hosts: all

  pre_tasks:
    - name: Update APT cache
      apt: update_cache=yes

  roles:
    - { role: "azavea.unzip" }
```

**Comment by hectcastro** [comment]
:+1:

**Pull request merged**